CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog for Rest-Framework-Auth-Toolkit
```

Comment:
This format is compatible with dependabot :fireworks: 

https://github.com/caravancoop/xpuni-be/pull/76